astropy/time/core.py

```diff
@@ -34,13 +34,6 @@
 __all__ = ['Time', 'TimeDelta', 'TIME_SCALES', 'TIME_DELTA_SCALES',
            'ScaleValueError', 'OperandTypeError']
 
-try:
-    # Not guaranteed available at setup time
-    from . import erfa_time
-except ImportError:
-    if not _ASTROPY_SETUP_:
-        raise
-
```

Comment:
@taldcroft - I assumed it was not a problem since the code as is already imports `astropy._erfa` [above] (
https://github.com/astropy/astropy/pull/4235/files#diff-ef1d13ad3ca4675e93a75fdf8461d387R19)